src/resources/parser/glb-parser.js

```diff
@@ -1297,6 +1269,28 @@ const createMaterial = function (gltfMaterial, textures, flipV) {
         material.diffuseVertexColor = false;
     }
 
+    // handle KHR_materials_specular absorption
+    if (gltfMaterial.hasOwnProperty('extensions') &&
+        gltfMaterial.extensions.hasOwnProperty('KHR_materials_specular')) {
+
+        const specularData = gltfMaterial.extensions.KHR_materials_specular;
+        if (specularData.hasOwnProperty('specularColorTexture')) {
+            material.f0Map = textures[specularData.specularColorTexture.index];
+            material.f0MapChannel = 'rgb';
```

Comment:
do we need the channel? Is there some other way this could be configured, or is rgb the only option here.